packages/api/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@comet-brevo-module/api",
```

Comment:
This should be `@comet/brevo-module-api` I suppose, or even `@comet/brevo-api`?

Reply:
I changed the name to `@comet/brevo-api` in 38a792aff39ac7019ec5c9155c85e173e3419c8b